compose-files/docker-compose.calendar.yml

```diff
@@ -15,7 +15,7 @@ services:
   ### Schul-Cloud Calendar
   ###
   calendar-postgres:
-    image: postgres:${POSTGRES_DOCKER_TAG:-alpine}
+    image: postgres:${POSTGRES_DOCKER_TAG:-latest}
```

Comment:
Would it make sense to limit this to the current major version instead of latest?

Reply:
the version we use is set be default.env, with the normal startup scripts it use our current version by default